app/lib/submission_builder/authentication_header.rb

```diff
@@ -7,6 +7,13 @@ def document
       build_xml_doc("AuthenticationHeader") do |xml|
         xml.FilingLicenseTypeCd "O"
         xml.FinancialResolution do
+          if @intake.has_banking_information_in_financial_resolution? && @intake.payment_or_deposit_type == "direct_deposit"
```

Comment:
thought of checking whether routing/account/signed at exists here (b/c all those are required if `FirstInput` is present -- but felt like `direct_deposit` check was enough to ensure those things would be present (routing/acct no) and we should have the esigned_at at this point